docs/src/pages/configurations/overview/index.md

```diff
@@ -0,0 +1,81 @@
+
+---
+id: 'overview'
+title: 'Configuration overview'
+---
+
+For CLI options see: [here](/docs/cli-options).
+
+> migration guide: This page documents the method to configure storybook introduced recently in 5.3.0, consult the [migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md) if you want to migrate to this format of configuring storybook.
+
+## main configuration
+
+Storybook has a few files it uses for configuration, and they are grouped together into a directory (default: `.storybook`).
+
+The most import file is the `main.js` file. This is where general config is declared.
+
+Here's an minimal example of a that file:
+
+```js
+module.exports = {
+  stories: ['../src/components/**/*.stories.js'],
+  addons: [
+    '@storybook/addon-essentials',
+  ],
+};
+```
+
+The addons field can refer to traditional [addons](docs/addons/introduction), but it can also include [presets](/docs/presets/introduction/) extending the config further.
+
+## manager & preview
+
+Storybook works by being split into 2 applications, which communicate with each other over a postmessage channel; called the "manager" and "preview".
```

Comment:
has "how to use postmessage channel to communicate with each other..." already been documented ? if yes, i'd be great to refer to, WDYT?

btw, In general it looks good to me :)

Reply:
Documenting storybook's general architecture is on my todo list, thanks for the review

Reply:
Let me know if I can help with that